Core/PathfinderBaseRules.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Immutable;
+
+namespace PCSharpGen.Core
+{
+    public class PathfinderBaseRules : BaseGameRules
```

Comment:
All this rule stuff is in a LST file, so we will need to translate those and read the lua in. This will probably have to be the first LUA processing we write.

Reply:
Yeah, that's fine, I just wanted to create something to play around with at first.  Totally make sense that we'd read these in from somewhere. :-)